pkg/cmdlets/commit/commit.go

```diff
@@ -167,5 +168,7 @@ func usernameForID(id strfmt.UUID, orgs []gmodel.Organization) (string, error) {
 		}
 	}
 
-	return "", locale.NewError("err_user_not_found", id.String())
+	placeholder := locale.Tl("deleted_username", "<deleted>")
+	logging.Debug("Could not determine username for commit author '%s'. Using placeholder value '%s'.", id, placeholder)
+	return placeholder, nil
```

Comment:
Doesn't look like this function ever returns an `error` anymore, so might as well drop that.

Reply:
Thanks, good catch.